src/reference/asciidoc/amqp.adoc

```diff
@@ -3789,119 +3853,189 @@ if its enclosing `ApplicationContext` is closed) it waits for in-flight messages
 Defaults to 5 seconds.
 After the limit is reached, if the channel is not transacted messages will be discarded.
 
+a| image::images/tickmark.png[]
+a| image::images/tickmark.png[]
+
 | txSize
 (transaction-size)
 
 | When used with `acknowledgeMode` AUTO, the container will attempt to process up to this number of messages before sending an ack (waiting for each one up to the receive timeout setting).
 This is also when a transactional channel is committed.
 If the `prefetchCount` is less than the `txSize`, it will be increased to match the `txSize`.
 
+a| image::images/tickmark.png[]
+a|
+
 | receiveTimeout
 (receive-timeout)
 
 | The maximum time to wait for each message.
 If acknowledgeMode=NONE this has very little effect - the container just spins round and asks for another message.
 It has the biggest effect for a transactional `Channel` with `txSize > 1`, since it can cause messages already consumed not to be acknowledged until the timeout expires.
 
+a| image::images/tickmark.png[]
+a|
+
 | autoStartup
 (auto-startup)
 
 | Flag to indicate that the container should start when the `ApplicationContext` does (as part of the `SmartLifecycle` callbacks which happen after all beans are initialized).
 Defaults to true, but set it to false if your broker might not be available on startup, and then call `start()` later manually when you know the broker is ready.
 
+a| image::images/tickmark.png[]
+a| image::images/tickmark.png[]
+
 | phase
 (phase)
 
 | When autoStartup is true, the lifecycle phase within which this container should start and stop.
 The lower the value the earlier this container will start and the later it will stop.
 The default is Integer.MAX_VALUE meaning the container will start as late as possible and stop as soon as possible.
 
+a| image::images/tickmark.png[]
+a| image::images/tickmark.png[]
+
 | adviceChain
 (advice-chain)
 
 | An array of AOP Advice to apply to the listener execution.
 This can be used to apply additional cross cutting concerns such as automatic retry in the event of broker death.
 Note that simple re-connection after an AMQP error is handled by the `CachingConnectionFactory`, as long as the broker is still alive.
 
+a| image::images/tickmark.png[]
+a| image::images/tickmark.png[]
+
 | taskExecutor
 (task-executor)
 
 | A reference to a Spring TaskExecutor (or standard JDK 1.5+ Executor) for executing listener invokers.
 Default is a SimpleAsyncTaskExecutor, using internally managed threads.
 
+a| image::images/tickmark.png[]
+a| image::images/tickmark.png[]
+
 | errorHandler
 (error-handler)
 
 | A reference to an ErrorHandler strategy for handling any uncaught Exceptions that may occur during the execution of the MessageListener.
 Default: `ConditionalRejectingErrorHandler`
 
+a| image::images/tickmark.png[]
+a| image::images/tickmark.png[]
+
+| consumersPerQueue
+(consumers-per-queue)
```

Comment:
I see `DMLC` `monitorInterval` and `taskScheduler` properties are missed in this table.
WDYT?